bims/models/biological_collection_cluster.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+"""Boundary model definition.
+"""
+
+from django.contrib.gis.db import models
+from bims.models.boundary import Boundary
+
+
+class BiologicalCollectionCluster(models.Model):
+    """Biological Cluster model."""
```

Comment:
We should better explain the intent here e.g.:

The cluster model creates pre-computer spatial cluster analysis for all sites and in a geographic region. The clustering is based on different geographical areas for different scales. E.g. small scales will cluster by country, larger scale by province, catchment etc. The cluster units are configurable based on the boundary model.